vmdb/app/models/classification.rb

```diff
@@ -310,12 +310,16 @@ def find_entry_by_name(name, region_id = self.my_region_number)
     tag.nil? ? nil : self.class.find_by_tag_id(tag.id)
   end
 
-  def self.find_by_name(name, region_id = self.my_region_number, ns = DEFAULT_NAMESPACE)
+  def self.find_tag_by_name(name, region_id = self.my_region_number, ns = DEFAULT_NAMESPACE)
     if region_id.nil?
-      tag = Tag.find_by_name(Classification.name2tag(name, 0, ns))
+      Tag.find_by_name(Classification.name2tag(name, 0, ns))
     else
-      tag = Tag.in_region(region_id).find_by_name(Classification.name2tag(name, 0, ns))
+      Tag.in_region(region_id).find_by_name(Classification.name2tag(name, 0, ns))
     end
+  end
```

Comment:
@tenderlove `name` is a property of the `Tag`, so yes, I guess I should move it there. Except the tag name is determined by code in Classification so.. it is a bit ambiguous.

I'll push a commit changing this and adding a test.